### ISSUE TITLE
Fragment syntax not supported

### ISSUE THREAD

**Issue by aryzing** [deankevorkian/language-typescript-react#1]
### Description

Fragments are not syntax-highlighted properly

### Steps to Reproduce

1. Write code to render a fragment:

```jsx
render() {
  return <>
    <div>Text</div>
  </>
}
```

**Expected behavior:** The fragment's opening tag characters `<` and `>` should appear with the same background as any other element.

**Actual behavior:** The `<` and `>` characters have a red background.

**Reproduces how often:** 100%

### Versions

```
Atom    : 1.25.1
Electron: 1.7.11
Chrome  : 58.0.3029.110
Node    : 7.9.0
```

### Additional Information

![image](https://user-images.githubusercontent.com/4924405/38730462-ed687266-3f16-11e8-9e58-738fb8410d85.png)

**Comment by deankevorkian** [comment]
Hey!
Thanks for reporting :). I should've already stated that this package is no longer maintained, because I created it in the first place since the [original language-typescript package](https://github.com/atom/language-typescript) had no support for React's TSX.
Since then, my bug report was handled [here](https://github.com/atom/ide-typescript/issues/15) so the current upstreamed language-typescript has all these features.
TL;DR - I should (really) remove this package, since everything it has is merged into language-typescript. So you should open a bug there instead, [and it appears there's an existing bug for this already](https://github.com/atom/language-typescript/issues/19).

Thanks and sorry for the confusion!
